src/main/java/com/example/demo/CardData/CardData.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.CardData;
```

Comment:
lets change project name to something different, this is not example demo

Reply:
done